src/integration-tests/certificates/certificates.spec.js

```diff
@@ -0,0 +1,16 @@
+const commonHelper = require('@commonTests')
+const schema = require('./responseSchema')
+jest.setTimeout(10000)
+
+describe('Certificate APIs ', function () {
+	let userDetails
+	beforeAll(async () => {
+		userDetails = await commonHelper.logIn()
+	})
+
+	it('List Certificate Templates', async () => {
+		let res = await request.get('/scp/v1/certificates/list').query({ page: 1, limit: 10 })
```

Comment:
can we have page and limit value in any common place? As it will be used through out the tests. @priyanka-TL